### ISSUE TITLE
Confirm change button color

### ISSUE THREAD

**Issue by mboskin** [covid-alert-ny/covid-green-app#97]
Confirm Changes button in the COVID Check in is a liter purple color but cannot be selected until you change a selection then button just slightly changes color.  When selected it changes to black color.  not sure if that is ok.
--

**Comment by AlanSl** [comment]
@mboskin Can you please clarify what the problem here is: is it that you find it unclear from the colour that the lighter purple button is disabled?

@stuartgiles Currently disabled buttons have the same styling as regular buttons but with 40% opacity, and active buttons take the outer background colour (shadow colour). If you want to change any of this ping me and I'll apply it

**Comment by mboskin** [comment]
Yes, it is unclear. Greater contrast is needed, especially for ADA compliance.

**Comment by mboskin** [comment]
The contrast needs to be higher. I commented this and reassigned it back.

_____________________

www.nystec.com<http://www.nystec.com/>

**Comment by ahurworth** [comment]
The styling of the Health Log's inactive state in build 17 has good contrast, but now doesn't meet the bar of perceivably inactive.  I have proposed an update to @stuartgiles .  Waiting for him to confirm the new updated colors and opacities.

**Comment by stuartgiles** [comment]
Sorry @AlanSl, I didn't realise this was waiting for my response.

In the current build, our disabled secondary button is dark grey text on a light grey background.
To increase the visual affordance that this button is now inactive we'd like to lighten the text colour to 
`#6E6E6E` or `RGB 110 110 110`.
This is lighter but still within acceptable contrast guidelines.

![Screenshot 2020-08-28 at 17 56 08](https://user-images.githubusercontent.com/400087/91593354-e7497900-e957-11ea-93f8-e92de7144fad.png)

Thanks

**Comment by floridemai** [comment]
This has been now addressed (in the next build) as per Stuart's comments.

**Comment by kevin-sheldon** [comment]
retested from my end and looks fine.

**Comment by mboskin** [comment]
Confirm Changes button still turns black upon successful use, still functions normally. Retest results:

Patrick Brennan | 9/1/20 4:14 PM | Fail | Confirm Changes button still turns black upon successful use, still functions normally.

**Comment by fahd-arshad** [comment]
@mboskin I am not sure what exactly the problem is here so I can't assign a priority to it. Please clarify exactly what needs to be done here and assign back to @helloantoine. There is nothing wrong with the actual state changing of the "Confirm changes" button; it should indeed stay disabled until a change is made, and then switch to enabled state. 

@helloantoine: I can not speak to the disabled state's color contrast, but what Stuart posted a few days earlier seems to address a secondary button's disabled state contrast, not a primary button's, which is the one "Confirm changes" is using. I am looking at it on iOS build 31 App Settings->My Health Log.

**Comment by floridemai** [comment]
This is indeed a bug. The "pressed" state of the button is black background with white text. The button does not go back to disabled state after press.

**Comment by floridemai** [comment]
@mboskin Black button state address in the next version.

**Comment by mboskin** [comment]
@floridemai What build/version number should be retested?

**Comment by floridemai** [comment]
@mboskin Will be after today's build. (I think that will be build 32)

**Comment by helloantoine** [comment]
Looks good to me.
